pttrack/models.py

```diff
@@ -329,6 +343,13 @@ def all_phones(self):
 
         return phones
 
+    def change_active_status(self):
+        ''' Will Activate or Inactivate the Patient'''
+        if self.needs_workup:
+            self.needs_workup = False
+        else:
+            self.needs_workup = True
```

Comment:
Purely a style thing, but I would have done
```python
self.needs_workup = not self.needs_workup
```

here, because it both denser and (IMHO) more clearly communicates the "toggle" nature of the function. I might even name the function `toggle_active_status`